docs/src/advanced/index.rst

```diff
@@ -0,0 +1,13 @@
+****************
+Advanced guides
```

Comment:
I don't fully agree with @jameshadfield here about the reuse of "Reference Guides" being confused with the main docs. In the context of the ncov RTD site, these are references materials and not how-to guides. If we were to pick one word out of "reference guides" to keep, it would be "reference".

But, unless the ncov reference guides appear in the same ToC as the main docs reference guides, I don't see any issue with a lower-level reference guides section specific to the ncov workflow.

Reply:
How about reference materials?

Reply:
I like that!